installer/tue-install-impl.bash

```diff
@@ -875,14 +910,18 @@ function tue-install-ros
     [ -n "$TUE_ROS_DISTRO" ] || tue-install-error "Environment variable 'TUE_ROS_DISTRO' is not set."
 
     local ros_pkg_name=${TUE_INSTALL_CURRENT_TARGET#ros-}
+    if [[ $ros_pkg_name == *-* ]]
+    then
+        tue-install-error "A ROS package cannot contain dashes (${ros_pkg_name}), rename to ros-${ros_pkg_name//-/_}"
```

Comment:
With the word package, @Timple 's argument still holds.
```suggestion
        tue-install-error "A ROS target cannot contain dashes (${ros_pkg_name}), rename to ros-${ros_pkg_name//-/_}"
```

Reply:
See new commit